src/main/java/mekanism/common/tile/TileEntityBoundingBlock.java

```diff
@@ -17,22 +17,32 @@
 import net.minecraft.util.math.BlockPos;
 import net.minecraftforge.common.capabilities.Capability;
 
+/**
+ * Multi-block used by wind turbines, solar panels, and other machines
+ */
 public class TileEntityBoundingBlock extends TileEntity implements ITileNetwork {
 
-    public BlockPos mainPos = BlockPos.ORIGIN;
+    private BlockPos _mainPos = BlockPos.ORIGIN;
```

Comment:
I am fine with the change of making this private, though I would rather the name remain `mainPos` rather than becoming `_mainPos`

Reply:
its a bit of a habit of mine to place _ in front of internal fields that should never be used outside set methods.